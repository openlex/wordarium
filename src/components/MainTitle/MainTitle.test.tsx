import React from "react";
import { shallow, ShallowWrapper } from "enzyme";
import { MainTitle } from "@components";
import renderer from "react-test-renderer";

describe("WordListItem markup", () => {
    let title: ShallowWrapper = {} as ShallowWrapper;
    const textValue = "text";

    beforeEach(() => {
        title = shallow(<MainTitle title={textValue} />);
    });

    it("Same title as in props", () => {
        expect(title.find('[data-test-id="value"]').text()).toEqual(textValue);
    });

    it("match snapshoot", () => {
        expect(
            renderer.create(<MainTitle title={textValue} />).toJSON()
        ).toMatchSnapshot();
    });
});
