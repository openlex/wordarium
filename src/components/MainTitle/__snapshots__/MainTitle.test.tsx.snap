// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WordListItem markup match snapshoot 1`] = `ShallowWrapper {}`;
